app/controllers/questions_controller.rb

```diff
@@ -10,28 +10,63 @@ def index
     @question_limitations = ENV["limit_question_index_to_users_questions_only"]
   end
 
+  def show
+    @questions = get_question_list()
+    @question_id = params[:id]
+    @question = @questions[(params[:id]).to_i - 1]["text"]
+
+    logger.debug "QUESTION's type in question's show: #{@questions[(params[:id]).to_i - 1].class}"
+
+    @answers = @questions[(params[:id]).to_i - 1]["answers"]
+    @answer = Answer.new
+  end
+
   def new
     @question = Question.new
     @question.answers.build
+    logger.debug "@question from new is #{@question}"
   end
 
   def create
     question = Question.new(params.require(:question)
       .permit(:text,
         answers_attributes:[:text]))
+
+    logger.debug "QUESTION's type in question's create: #{question.class}"
+
     question.user_id = current_user.id
     question.post_question(question)
+
+    logger.debug "@QUESTION's type in question's destroy: #{@question.class}"
+
     redirect_to new_question_path, notice: "Your question has been submitted! Enter another if you would like."
   end
 
-  def show
+  # def show
+  #   @questions = get_question_list()
+  #   @question_id = params[:id]
+  #   @question = @questions[(params[:id]).to_i - 1]["text"]
+  #   @answers = @questions[(params[:id]).to_i - 1]["answers"]
+  #   @answer = Answer.new
+  # end
+
+
+  def destroy
+    logger.debug "PARAMS in question's destroy: #{params}"
```

Comment:
I think the Rails default server logs give you the params when a method is called. E.g.:
```
Processing by QuestionsController#destroy as HTML
  Parameters: {"authenticity_token"=>"...", "answers"=>[{"created_at"=>"2015-04-24T21:11:18.029Z", "id"=>"15", "question_id"=>"6", "text"=>"ok", "updated_at"=>"2015-04-24T21:11:18.029Z"}], "created_at"=>"2015-04-24T21:11:18.011Z", "text"=>"new one", "updated_at"=>"2015-04-24T21:11:18.011Z", "user_id"=>"5", "id"=>"6"}
```